commands/deploy.go

```diff
@@ -249,7 +250,34 @@ Error: %s`, fprocessErr.Error())
 				function.ReadOnlyRootFilesystem = deployFlags.readOnlyRootFilesystem
 			}
 
-			statusCode := proxy.DeployFunction(function.FProcess, services.Provider.GatewayURL, function.Name, function.Image, function.RegistryAuth, function.Language, deployFlags.replace, allEnvironment, services.Provider.Network, functionConstraints, deployFlags.update, deployFlags.secrets, allLabels, annotations, functionResourceRequest1, function.ReadOnlyRootFilesystem, tlsInsecure)
+			// Need to alter Gateway to allow nil/empty string as fprocess, to avoid this repetition.
+			var fprocessTemplate string
+			if len(function.FProcess) > 0 {
+				fprocessTemplate = function.FProcess
+			}
+			req := requests.CreateFunctionRequest{
```

Comment:
Is this PR also covering #475 too?

Reply:
Oh, I suppose it is! 😅 I'm very obsessive-compulsive in a way with regards to my personal code-style. If I need to push anything else to cover that PR as well, I'll take the notes down and get right to it. 👍